assets/sass/patterns/atoms/list.scss

```diff
@@ -18,3 +18,7 @@ $prefixes: ("alpha-lower": "lower-latin", "alpha-upper": "upper-latin", "bullet"
     }
   }
 }
+
+.list--teaser {
+  @include padding($blg-space-small-in-px, "top");
```

Comment:
I have no objection to that approach but @davidcmoulton didn't seem keen on the idea. I can confirm that wrapping the text in a `<p>` element gives the text and list the separation that it needs.